src/web/content/static/css/letterpress.css

```diff
@@ -572,7 +576,7 @@ a.sms-large-font:hover {
 
 .sms-sort-container {
     float: right;
-    width: 28rem;
+    width: auto;
```

Comment:
Removed the fixed width for the filters drop-down. The auto is necessary because Bootstrap already sets a 100% width on this, that's why I had a fixed width here in the first place.